app/assets/stylesheets/application.css.scss

```diff
@@ -39,6 +46,13 @@ body {
   background-position: center center;
 }
 
+.application-body-armory{
+  background-image: url('armory.jpg');
+  background-size:     cover;
```

Comment:
Properties should be ordered background-image, background-position, background-repeat, background-size<br>Colon after property should be followed by one space